warp/cairo-src/evm/calls.cairo

```diff
@@ -56,10 +56,10 @@ func calldata_load{range_check_ptr, exec_env : ExecutionEnvironment*}(offset) ->
     return (value=value)
 end
 
-func returndata_copy{range_check_ptr, exec_env : ExecutionEnvironment, memory_dict : DictAccess*}(
+func returndata_copy{range_check_ptr, exec_env : ExecutionEnvironment*, memory_dict : DictAccess*}(
         memory_pos : Uint256, returndata_pos : Uint256, length : Uint256):
     array_copy_to_memory(
-        exec_env.returndata_len, exec_env.returndata, returndata_pos, memory_pos, length)
+        exec_env.returndata_len, exec_env.returndata, returndata_pos.low, memory_pos.low, length.low)
```

Comment:
Let's do it in a separate commit and add a test to ensure that desync doesn't happen again